hack/test-integration.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+ORIGIN_VERSION=latest
+if [[ -n "$PULL_BASE_REF" && $PULL_BASE_REF != "master" ]]; then
```

Comment:
eek hate to leak ci into repos...

Reply:
@stevekuznetsov sure. i'm open to suggestions......

Reply:
this logic in aos-cd-jobs?

Reply:
the script in the repo takes as input the version it needs?